drivers/storage/portworx/deployment.go

```diff
@@ -878,6 +895,10 @@ func (t *template) getEnvList() []v1.EnvVar {
 
 func (t *template) getVolumeMounts() []v1.VolumeMount {
 	volumeInfoList := append([]volumeInfo{}, defaultVolumeInfoList...)
+	if t.isK3s {
+		volumeInfoList = append(volumeInfoList, t.getK3sVolumeInfoList()...)
+	}
+
 	volumeMounts := make([]v1.VolumeMount, 0)
```

Comment:
nit -- the following allocation is a little bit better:
```go
volumeMounts := make([]v1.VolumeMount, 0, len(volumeInfoList))
```
* I normally always take advantage of the fact that we know exactly how large will the produced array be
* if we leave it up to `append()` to extend the array-capacity, then it'll _double_ the array capacity (also _copy_ all the elements) every time it extends the capacity